lib/model/unreads.dart

```diff
@@ -91,6 +96,9 @@ class Unreads extends ChangeNotifier {
   /// Unread DM messages, as: DM narrow → message IDs (sorted).
   final Map<DmNarrow, QueueList<int>> dms;
 
+  // Maps lowercase topic names for lookup to one of the variants of the topic (case preserving).
+  final Map<String, TopicName> topicMapper;
```

Comment:
This won't have the right behavior because this map is shared between channels. So if there's one conversation [at](https://en.wikipedia.org/wiki/Ios) `#islands > Ios` and another at `#mobile > iOS`, then one or the other would get wrongly rendered with the other's casing.